share/lib/python/neuron/rxd/node.py

```diff
@@ -65,12 +53,23 @@ def _allocate(num):
 def _remove(start, stop):
     """ delete old volumes, surface areas and diff values in from global arrays
     """
-    global _volumes, _surface_area, _diffs, _states
+    global _volumes, _surface_area, _diffs, _states, _node_fluxes, _has_node_fluxes
     #Remove entries that have to be recalculated
-    _volumes = numpy.delete(_volumes,list(range(start,stop)))
-    _surface_area = numpy.delete(_surface_area,list(range(start,stop)))
-    _diffs = numpy.delete(_diffs,list(range(start,stop)))
-    _states = numpy.delete(_states,list(range(start,stop)))
+    dels = list(range(start,stop))
+    _volumes = numpy.delete(_volumes, dels)
+    _surface_area = numpy.delete(_surface_area, dels)
+    _diffs = numpy.delete(_diffs, dels)
+    _states = numpy.delete(_states, dels)
+
+    # remove _node_flux
+    newflux =  {'index': [], 'type': [], 'source': [], 'scale': [], 'region': []} 
+    for (i,idx) in enumerate(_node_fluxes['index']):
+        if idx not in dels:
```

Comment:
it is not very efficient to look in `dels` when you can simply compare `start <= idx < stop`

Reply:
With respect to
using namespace std;
in ivstream.h. On removing that, I suddenly had the issue of hundreds of errors with regard to ostream when building nrn. (note nrn/src/ivos/ivstream.h is now a copy of iv/src/include/ivstream.h). What is a bit puzzling to me at the moment is why there is no
problem with the many ostream when building iv.  As an aside. I realize that the travis build for iv needs to also try a build of nrn to make sure the change to iv did not break the nrn build (as happened with the merge of the cmake pull request for iv). Though I can imagine that could introduce a tricky chicken/egg problem.

Reply:
@nrnhines : sorry, didn't see this message. When removing the `using namespace std` you have to make sure that all names that come from `std::` but were being used without their proper namespace need to have that namespace prepended, i.e. all your `ostream` must be written now as `std::ostream`.